index.js

```diff
@@ -0,0 +1,91 @@
+// toggle
+
+const Humb = document.querySelector('#humb');
+const Menu = document.querySelector('.nav-ul');
+
+Humb.addEventListener('click', () => {
+  Humb.classList.toggle('active');
+  Menu.classList.toggle('active');
+});
+const teamSection = document.querySelector('.team-cont');
+const person = [
+
+  {
+    title: 'reagan clinton',
+    titleDetail: 'senior software engeneer',
+    description: 'Prepares and installs solutions by determining and designing system specifications, standards, and programming',
+    imageSrc: './images/pexels-pixabay-354951.jpg',
+    imageAlt: 'first person',
+  },
+  {
+    title: 'reagan clinton',
+    titleDetail: 'senior software engeneer',
+    description: 'Prepares and installs solutions by determining and designing system specifications, standards, and programming',
+    imageSrc: './images/pexels-pixabay-354951.jpg',
+    imageAlt: 'first person',
+  },
+  {
+    title: 'reagan clinton',
+    titleDetail: 'senior software engeneer',
+    description: 'Prepares and installs solutions by determining and designing system specifications, standards, and programming',
```

Comment:
- [ ] kindly make some changes here even if its just with the name so we know they are getting all the different objects from the array